mora/app.py

```diff
@@ -52,6 +54,6 @@ def handle_invalid_usage(error):
 @app.route('/<path:path>')
 def root(path=''):
     if path.split('/', 1)[0] == 'service':
-        raise exceptions.NotFoundError('no such endpoint')
+        raise exceptions.NotFoundError(ErrorCodes.E_NO_SUCH_ENDPOINT)
```

Comment:
…as an example, we now have some duplication between the exception object and the enum here — and the exception doesn't really add anything.